test/test_indexing.py

```diff
@@ -336,20 +318,6 @@ def test_int_assignment(self, device):
         x[1] = torch.arange(5, 7, device=device)
         self.assertEqual(x.tolist(), [[0, 1], [5, 6]])
 
-    def test_byte_tensor_assignment(self, device):
```

Comment:
It does not look like there is similar test with bool_tensor_assignment, so please modify this test to use bool tensors